src/index.js

```diff
@@ -223,7 +223,18 @@ function querySelectorAll (selector, context = document) {
   return query(selector, context, true)
 }
 
+function getElementsByClassName (classNames, context = document) {
+  const oneSpaceSeparator = classNames.trim().replace(/\s\s+/g, ' ');
+  const classNamesSplit = oneSpaceSeparator.split(' ');
+  let selector = "";
+  for (let i = 0; i < classNamesSplit.length; i++) {
+    selector += '.' + classNamesSplit[i];
+  }
+  return query(selector, context, true);
```

Comment:
Technically, `getElementsByClassName` is [supposed](https://developer.mozilla.org/en-US/docs/Web/API/Element/getElementsByClassName) to return a live-updating HTMLCollection. Should we support this?

Reply:
kagekiri is stateless right? so wouldn't it be up to the consumer to implement that?

Reply:
kagekiri is stateless, yes. I'm just calling out that if we're aiming for 1:1 compatibility with `getElementsByClassName`, then we would need to implement a live HTMLCollection.